guacamole/src/main/frontend/src/app/clipboard/directives/guacClipboard.js

```diff
@@ -47,6 +47,12 @@ angular.module('clipboard').directive('guacClipboard', ['$injector',
     config.controller = ['$scope', '$injector', '$element',
             function guacClipboardController($scope, $injector, $element) {
 
+        /**
+         * Set to true if the clipboard data should be displayed in the  
+         * Clipboard Editor. Otherwise, false.
+         */
+        var shouldDisplay = false;
```

Comment:
To match the existing code style of other variables declared here, this should have a `@type` annotation.